translate/canon/ListLanguageNames.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START translate_list_language_names]
```

Comment:
Maybe worth adding a standard disclaimer about our snippets style in every file containing C# snippets - saying that every method is self contained and code duplication between different files of the same partial class was done for educational purposes.